db/migrate/20191025230639_change_donation_money_raised_type_to_float.rb

```diff
@@ -0,0 +1,13 @@
+class ChangeDonationMoneyRaisedTypeToFloat < ActiveRecord::Migration[6.0]
+  def change
+    reversible do |change|
+      change.up do
+        change_column :donations, :money_raised, :decimal, scale: 2
```

Comment:
I think money is currently stored in cents - do we need to add x100 and /100 updates to the migration?